src/components/index/page-sections/CallToActionSection.jsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from '@reach/router';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import CallToActionLogo from '../../../images/call-to-action.png';
+import IndexGradientBorderButtonBasic from '../../buttons/gradient-border-buttons/IndexGradientBorderButtonBasic';
+
+const CallToActionSection = () => {
+  return (
+    // Section Container
+    // <div className="container mx-auto grid grid-cols-2 gap-10 mb-40">
+    <div className="container mx-auto flex flex-wrap flex-row mb-40">
+      {/* Image */}
+      <div className="flex flex-col w-full md:w-1/2 px-4 ">
+        <img src={CallToActionLogo} alt="" className="object-scale-down" />
+      </div>
+      {/* Content */}
+      <div className="flex flex-col w-full md:w-1/2 justify-center items-center ">
+        <span className="calltoaction-text my-5 text-center w-full md:w-1/2">
```

Comment:
@ndrslmpk versuch bitte klassen genereisch zu halten.

`calltoaction-text` ist mmn zu spezifisch. Generell ist das eine headline. (gleicher style wie z.B. Product Boxes). Du könntest also Klassen für h1, h2, h3 etc. styles anlegen und die dann an der Stelle hinzufügen.

Reply:
Ist dem so? Ich dachte nämlich, dass h1 etc. nicht passen, weil sie ja den Content strukturieren würden. In dem Fall ist es jedoch keine Headline und somit mMn als "spezielles Element" anzusehen, sodass ich dafür ein extra css prop angelegt habe.

Reply:
yeah sorry, hab mich nicht klar genug ausgedrückt.

Ich persönlich definiere immer .h1 .h2 .h3 etc. als CSS Klassen. 
Die Designs sind ja danach auch angelehnt. 

somit kannst du auch an einem `<span className="h1">` nutzen, um die gleichen styles zu haben. 
Aber du hast recht -> ein html h2 tag sollte hier nicht hin